app/src/main/java/org/mozilla/focus/widget/BrowserNavigationOverlay.kt

```diff
@@ -72,7 +72,6 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         get() = Settings.getInstance(context).isBlockingEnabled
         set(value) {
             Settings.getInstance(context).isBlockingEnabled = value
```

Comment:
Since you're removing the other line you can do a `set(value) = Settings...` :P

Reply:
I actually can't because it's not returning a value. Nice try though. ;)